library/rsa.c

```diff
@@ -345,7 +329,7 @@ int mbedtls_rsa_validate_crt( const mbedtls_mpi *P,  const mbedtls_mpi *Q,
         }
     }
 
-    /* Check that DQ - Q == 0 mod Q - 1 */
+    /* Check that DQ - D == 0 mod Q - 1 */
```

Comment:
Here and elsewhere, we're calculating A mod B (which also calculates the quotient under the hood) in order to test whether A is divisible by B. Is there a more efficient way?

Reply:
@gilles-peskine-arm Not that I'm aware of.